content/minfraud/release-notes/2024.md

```diff
@@ -9,6 +9,23 @@ outputs: ['html', 'rss']
 Subscribe to the [minFraud release notes RSS feed](/minfraud/release-notes/rss.xml).
 {{</ alert >}}
 
+{{< release-note date="2024-10-22" title="Deprecation of risk factor scores/subscores" >}}
+
+On Monday, November 4, 2024, we will be deprecating the risk factor scores/
+subscores data in the minFraud Factors service. Risk factor scores/subscores 
+will be removed from the API response in March, 2025.
```

Comment:
Do we need the comma after `March` if we aren't putting in a day?

Reply:
i added it because i liked how it anchored the text so just going to leave it, but good callout!